paddle/fluid/inference/tests/infer_ut/test_LeViT.cc

```diff
@@ -174,6 +174,10 @@ TEST(tensorrt_tester_LeViT, multi_thread4_trt_fp32_bz2) {
 
 int main(int argc, char** argv) {
   ::testing::InitGoogleTest(&argc, argv);
+#ifdef GFLAGS_NAMESPACE
+  GFLAGS_NAMESPACE::ParseCommandLineFlags(&argc, &argv, true);
+#else
```

Comment:
```suggestion
::GFLAGS_NAMESPACE::ParseCommandLineFlags(&argc, &argv, true);
```
直接写成这样就可以了，因为GFLAGS_NAMESPACE默认定义就是`google`。其他的地方也是类似。

Reply:
Done